Modules/_pickle.c

```diff
@@ -6623,13 +6623,9 @@ _pickle_Unpickler_find_class_impl(UnpicklerObject *self,
         }
     }
 
-    module = PyImport_GetModule(module_name);
+    module = PyImport_Import(module_name);
```

Comment:
@tjb900 Can you add a comment explaining why we don't go through the `PyImport_GetModule` shortcut? Otherwise we're bound to make the same mistake again.